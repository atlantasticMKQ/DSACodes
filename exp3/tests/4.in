{{{{},59,{{{{{{{},40,{{},4,{{{},51,{{},18,{}}},24,{{{},12,{{{},48,{{},74,{{{{},42,{{},13,{{},75,{{{{},90,{{{},7,{{{{{},65,{{{{},16,{}},78,{{},54,{}}},69,{}}},50,{{{},87,{}},55,{{},5,{}}}},56,{{},86,{}}},15,{{},64,{}}}},39,{{},9,{}}}},89,{}},19,{{},79,{}}}}}},45,{{},61,{}}},57,{{{{},76,{}},98,{{},17,{}}},11,{{{},67,{}},32,{}}}}}},6,{{},37,{{},71,{{},58,{}}}}}},73,{{{{},70,{{},23,{{},93,{}}}},68,{{{},82,{}},95,{}}},100,{{{{},26,{{},63,{}}},49,{{{},33,{}},8,{}}},92,{{},99,{{{},14,{}},53,{{},80,{}}}}}}}}}},66,{{{{},20,{}},34,{{{},1,{{},83,{}}},22,{}}},47,{{},97,{}}}},81,{{{{},31,{{},41,{{{},28,{}},43,{}}}},10,{{{},85,{{},62,{}}},25,{{{},46,{}},96,{{},36,{}}}}},72,{{{},38,{}},52,{{},44,{{{},94,{}},27,{{},77,{}}}}}}},30,{}},91,{}},3,{}}},88,{{},2,{{},35,{}}}},21,{{{},84,{{},29,{}}},60,{}}}
