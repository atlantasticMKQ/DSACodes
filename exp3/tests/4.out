21,88,59,3,91,30,81,66,40,4,24,51,18,73,12,6,48,74,57,45,42,13,75,19,89,90,39,7,15,56,50,65,69,78,16,54,55,87,5,86,64,9,79,61,11,98,76,17,32,67,37,71,58,100,68,70,23,93,95,82,92,49,26,63,8,33,99,53,14,80,47,34,20,22,1,83,97,72,10,31,41,43,28,25,85,62,96,46,36,52,38,44,27,94,77,2,35,60,84,29
21,88,60,59,2,84,3,35,29,91,30,81,66,72,40,47,10,52,4,34,97,31,25,38,44,24,20,22,41,85,96,27,51,73,1,43,62,46,36,94,77,18,12,100,83,28,6,68,92,48,37,70,95,49,99,74,71,23,82,26,8,53,57,58,93,63,33,14,80,45,11,42,61,98,32,13,76,17,67,75,19,89,79,90,39,7,9,15,56,64,50,86,65,55,69,87,5,78,16,54
12
9
33